src/vm/functions/principals.rs

```diff
@@ -0,0 +1,51 @@
+use vm::costs::cost_functions::ClarityCostFunction;
+use vm::costs::{cost_functions, runtime_cost, CostTracker};
+use vm::errors::{
+    check_argument_count, CheckErrors, Error, InterpreterError, InterpreterResult as Result,
+    RuntimeErrorType,
+};
+use vm::representations::SymbolicExpression;
+use vm::types::{
+    PrincipalData, QualifiedContractIdentifier, StandardPrincipalData, TypeSignature, Value,
+};
+use vm::{eval, Environment, LocalContext};
+
+use vm::database::ClarityDatabase;
+use vm::database::STXBalance;
+
+use chainstate::stacks::{
+    C32_ADDRESS_VERSION_MAINNET_MULTISIG, C32_ADDRESS_VERSION_MAINNET_SINGLESIG,
+    C32_ADDRESS_VERSION_TESTNET_MULTISIG, C32_ADDRESS_VERSION_TESTNET_SINGLESIG,
+};
+
+pub fn special_is_standard(
+    args: &[SymbolicExpression],
+    env: &mut Environment,
+    context: &LocalContext,
+) -> Result<Value> {
+    check_argument_count(1, args)?;
+    runtime_cost(ClarityCostFunction::StxTransfer, env, 0)?;
```

Comment:
This is not the correct cost function to use here, since this is a Stacks 2.1 method.

Reply:
which cost function would you use?

Reply:
Once #2751 merges, we should use Unimplemented here.